share/completions/gpg.fish

```diff
@@ -72,6 +73,69 @@ function __fish_print_gpg_algo -d "Complete using all algorithms of the type spe
     }'
 end
 
+if test (gpg --version | sed '1q;d' | cut -b 13) -eq 2
```

Comment:
Rather than shelling out to two external commands, let's use internal commands:

`gpg --version | string match -q 'gpg (GnuPG) 2.*'`

(actually it might be better to invert the test to match GPG 1.x)

Reply:
I've implemented this change.